docs/xxcompactstrings.md

```diff
@@ -40,4 +40,6 @@ Further details are available at [JEP 254: Compact Strings](https://openjdk.org/
 | `-XX:+CompactStrings`    | Enable `String` compression  |         |
 | `-XX:-CompactStrings`    | Disable `String` compression | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
```

Comment:
It's still marked as disabled by default in the table, this should be updated.

Reply:
But, it is enabled by default only for Version 11 and later, right? So for version 8 and other versions before 11, It is still disabled, by default, isn't it?

Reply:
Can we reflect that in the table with an image for `jdk11+`? I think we've done that elsewhere. 
There are no other versions than 8 before 11.

Reply:
So, should I give tick marks in both the rows but with different icons - 1st row with jdk11+ icon and second row with jdk8 icon?

Reply:
Yes